sos/report/plugins/networking.py

```diff
@@ -82,7 +82,10 @@ def setup(self):
 
         self.add_cmd_output("ip -o addr", root_symlink="ip_addr",
                             tags='ip_addr')
-        self.add_cmd_output("route -n", root_symlink="route", tags='route')
+        self.add_cmd_output("route -n -4", root_symlink="route",
+                            tags='route')
```

Comment:
Ack to making this explicitly `-n -4`. For the tags, let's make that `tags=['route', 'route_v4']` as well. We keep current automation happy, and provide a mechanism to be more exact in the future.